lib/associations/base.js

```diff
@@ -1,6 +1,13 @@
 'use strict';
+const AssociationError = require('./../errors').AssociationError;
 
-var Association = function() {};
+var Association = function(source, options) {
+  if (source.hasAlias(options.as)) {
+    throw new AssociationError(`You have used the alias ${options.as} in two separate associations. ` +
+    `Aliased associations must have unique aliases.`
```

Comment:
You can use template strings when there are expressions inside it. The last string is just a simple string